client/js/socket-events/uploads.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const $ = require("jquery");
+const socket = require("../socket");
+const wrapCursor = require("undate").wrapCursor;
+
+socket.on("upload:success", (url) => {
+	const fullURL = new URL(url, location);
```

Comment:
Let's future proof this and check if `url` starts with `http`. If it does then it's a full url.